AdaptivePELE/freeEnergies/estimateDGAdaptive.py

```diff
@@ -7,20 +7,20 @@
 from AdaptivePELE.freeEnergies import estimateDG
 
 
-def main(trajsPerEpoch, lagtime, nclusters, clusteringStride=1, nruns=10, lagtimes=[1, 10, 25, 50, 100, 250, 400, 500, 600, 1000]):
+def main(trajsPerEpoch, lagtime, nclusters, clusteringStride=1, nruns=10, lagtimes=[1, 10, 25, 50, 100, 250, 400, 500, 600, 1000], output=False, only_last=False):
     allFolders = np.array(glob.glob("MSM_*"))
     epochs = [int(folder[4:]) for folder in allFolders]
     args = np.argsort(epochs)
     sortedFolders = allFolders[args]
-    origDir = os.getcwd()
+    origDir = output if output else os.getcwd()
     resultsFile = os.path.join(origDir, "results.txt")
 
     with open(resultsFile, "a") as f:
         f.write("#Epoch DG StdDG Db StdDb\n")
         f.write("#=======================\n")
 
     resultsEpoch = []
-    initialEpoch = 0
+    initialEpoch = -1 if only_last else 0
     for i, folder in enumerate(sortedFolders[initialEpoch:]):
         epoch = i + initialEpoch
```

Comment:
This line needs to change, if initialEpoch is -1, then the value of ntrajs in line 28 would be 0.